libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Model/SFSDKInstrumentationEvent.m

```diff
@@ -133,6 +133,56 @@ - (instancetype) initWithJson:(NSData *) jsonRepresentation {
     return self;
 }
 
+#pragma mark - NSCopying
+
+- (instancetype)copyWithZone:(NSZone *)zone {
```

Comment:
This is the only new method here.  The other changes in this file are just moving methods around to group them.